public/account.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="ru">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Spotify</title>
+        <link rel="stylesheet" href="../src/index.css"/>
```

Comment:
Не корректный путь
css должен находиться рядом с html файлом
Ссылка должна быть вида:
```suggestion
        <link rel="stylesheet" href="%PUBLIC_URL%/src/index.css"/>
```
Также плохая практика не использовать фавиконку на странице

это относится и к другим местам